### PR TITLE
[Breaking change] Refactor typography and add FinePrint

### PR DESCRIPTION
**Pull request by fabien0102**
### Summary

- remove Heading1
- remove Heading2
- add a real styleguidist section for typography
- use actual theme constant for typography components
- add FinePrint component
- use theme constant into CardColumn

### Related issue

https://trello.com/c/OKIZxik8

### To be tested

Fabien
- [x] No error/warning in the console

Tester 1
- [x] The netlify build is working
- [x] The documentation is beautiful with sections
- [x] All typography are correct regarding the design

Tester 2
- [ ] The netlify build is working
- [ ] The documentation is beautiful with sections
- [ ] All typography are correct regarding the design

### CODE REVIEW

File: styleguide/SectionHeadingRenderer.tsx [hunk 1]
```diff
@@ -2,15 +2,12 @@ import * as React from "react"
...
+  depth: number
 }
 
-const SectionHeadingRenderer: React.SFC<SectionHeadingRendererProps> = ({ children, toolbar, id }) => (
-  <CardHeader id={id} action={toolbar}>
-    {children}
-  </CardHeader>
-)
+const SectionHeadingRenderer: React.SFC<SectionHeadingRendererProps> = ({ children, toolbar, depth }) =>
+  depth > 1 ? <CardHeader action={toolbar}>{children}</CardHeader> : null
```
Comment (TejasQ) [inline styleguide/SectionHeadingRenderer.tsx]: This can use a logical and operator to be more concise.

**Comment by TejasQ** [comment]
@micha-f is the [typography correctly representative of the design](https://deploy-preview-651--operational-ui.netlify.com/#Body)? (Scroll to the bottom)

File: styleguide/SectionHeadingRenderer.tsx (hunk 1 above)
Comment (peterszerzo) [reply styleguide/SectionHeadingRenderer.tsx]: Isn't returning `undefined` from a render method illegal in React? I used to get warnings.

**Comment by micha-f** [comment]
Can we use `line-height: 1.4` for text?

**Comment by TejasQ** [comment]
Looks good. If @micha-f says the text looks file on the deploy preview, merge it.

**Comment by micha-f** [comment]
beautiful @fabien0102

**Pull request merged**